.github/labels.yaml

```diff
@@ -69,6 +69,10 @@
   color: 3DED97
   description: Label to trigger Github Action tests.
 
+- name: 'docs: deploy preview'
```

Comment:
super nit: it may be more common to not use spaces when using a label to trigger an action ie `docs:deploy-preview`. Other previous art is `kokoro:force-run`, `kokoro:run`, `release-please:force-run`